Jellyfin.Plugin.Tvdb/Configuration/PluginConfiguration.cs

```diff
@@ -1,4 +1,4 @@
-﻿using MediaBrowser.Model.Plugins;
+using MediaBrowser.Model.Plugins;
```

Comment:
This file's changes should be reverted, I will include the api key in a separate PR

Reply:
How about I change it to String.Empty and you update the API key accordingly?